ctapipe/tools/tests/test_merge.py

```diff
@@ -113,7 +113,8 @@ def test_allowed_tels(tmp_path, dl1_file, dl1_proton_file):
             str(dl1_file),
             str(dl1_proton_file),
             f"--output={output}",
-            "--allowed-tels=[1,2]",
+            "--allowed-tels=1",
+            "--allowed-tels=2",
```

Comment:
maybe I lost some previous discussion: why is this needed?

Reply:
It generated traitlets warnings. Passing a list literal is deprecated and will be removed soon